core/src/main/java/nl/nn/adapterframework/jdbc/FixedQuerySender.java

```diff
@@ -65,8 +66,8 @@ protected String getQuery(Message message) {
 
 	@Override
 	public boolean canProvideOutputStream() {
-		return  "updateClob".equalsIgnoreCase(getQueryType()) && StringUtils.isEmpty(getClobSessionKey()) ||
-				"updateBlob".equalsIgnoreCase(getQueryType()) && StringUtils.isEmpty(getBlobSessionKey());
+		return QueryType.UPDATECLOB==getQueryTypeEnum() && StringUtils.isEmpty(getClobSessionKey()) ||
+				QueryType.UPDATEBLOB==getQueryTypeEnum() && StringUtils.isEmpty(getBlobSessionKey());
```

Comment:
When you compare strings to a fixed value, usually the fixed value is on the left side: ``"updateClob".equalsIgnoreCase(getQueryType())``
When testing enums, it is easier to read it the other way round:
 `` getQueryTypeEnum()==QueryType.UPDATECLOB``